arviz/plots/backends/bokeh/essplot.py

```diff
@@ -88,7 +88,7 @@ def plot_ess(
 
             values = data[var_name].sel(**selection).values.flatten()
             mask = idata.sample_stats[rug_kind].values.flatten()
-            values = rankdata(values)[mask]
+            values = rankdata(values, method="average")[mask]
```

Comment:
method="average" is the default, right? So this is making it explicit, right?

Reply:
yes, nothing really changes